geonode/management_commands_http/forms.py

```diff
@@ -0,0 +1,42 @@
+import json
+
+from django import forms
+
+from geonode.management_commands_http.models import ManagementCommandJob
+from geonode.management_commands_http.utils.commands import (
+    get_management_commands,
+)
+
+
+class ManagementCommandJobAdminForm(forms.ModelForm):
+    autostart = forms.BooleanField(required=False)
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.available_commands = get_management_commands()
+        self.fields["command"] = forms.ChoiceField(
+            choices=[(command, command) for command in self.available_commands]
+        )
+
+    class Meta:
+        model = ManagementCommandJob
+        fields = ("command", "args", "kwargs",)
+
+    def clean_args(self):
+        value = self.cleaned_data.get('args')
+        value_json = json.loads(value)
```

Comment:
Consider to use [ast.literal_eval](https://docs.python.org/3/library/ast.html#ast.literal_eval) instead of `json.loads`:
```python
Example 1:
input = '["acb"]'
json.loads -> ['acb']
ast.literal_eval -> ['acb']

Example2: input = "['acb']"
json.loads -> json.decoder.JSONDecodeError: Expecting value: line 1 column 2 (char 1)
ast.literal_eval -> ['acb']
```

Reply:
* `ast.literal_eval` has some security issues, it opens the possibility of adding a malicious code. 
* In Django 3 (newer versions), the args and kwargs can be stored using a JSONField. During the port to mast there will be no need for json.loads and json.dumps because the JSONField will handle it.
* An example is the celery-beat scheduler in django admin (http://localhost:8000/en/admin/django_celery_beat/periodictask/add/). I think instead os using `ast.literal_eval` we communicate better with the user adding a help message to django admin and in documentation. "JSON encoded positional arguments (Example: ["arg1", "arg2"])"
![image](https://user-images.githubusercontent.com/12943132/135083929-b1a999a2-8acf-4af6-96e1-6703e78b89c6.png)

Reply:
The help message is a good solution :)